junit-runtime/src/main/scala/org/scalajs/junit/ScalaJSJUnitTest.scala

```diff
@@ -0,0 +1,10 @@
+package org.scalajs.junit
+
+import scala.scalajs.js.annotation.JSExportDescendentObjects
+import scala.scalajs.js.annotation.JSExportDescendentClasses
+
+@JSExportDescendentObjects
+@JSExportDescendentClasses
```

Comment:
I don't think these should be there. We don't export the test classes, since they're instantiated through the `newInstance` method of the metadata.